docs/quickstart.md

```diff
@@ -29,7 +29,8 @@ $ redskyctl init
 Generally you will want to write your own experiments to run trials on your own applications. For the purposes of this guide we can use the simple example found in the `redskyops-recipes` [repository on GitHub](https://github.com/redskyops/redskyops-recipes/tree/master/simple):
 
 ```sh
-$ kustomize build github.com/redskyops/redskyops-recipes//simple | kubectl apply -f -
+$ git clone https://github.com/redskyops/redskyops-recipes.git
+$ kustomize build redskyops-recipes/simple | kubectl apply -f -
```

Comment:
We had switched this explicitly to the remote build form to prevent the need for the explicit clone step in the examples (though the double slashes probably aren't necessary).

Reply:
Also, like `tutorial`, the `simple` recipe has been removed.

Reply:
let's have quickstart point to the postgres recipe